lib/game.rb

```diff
@@ -0,0 +1,113 @@
+class Game
+  def initialize(score_limit = 6)
+    @player1 = nil
+    @player2 = nil
+    @encrypter = nil
+    @hacker = nil
+    @current_set = nil
+    @sets = []
+    @render = Render.new
+    @score_limit = score_limit
+    @is_pair = true
```

Comment:
"pair" means "pareja", not "par". The right translation is "even" (and "odd" for "impar")
```suggestion
    @is_even = true
```

Reply:
> Also I think you could replace this variable with a method Game#even_round? that determines if the round is even based on the @sets

I think that too, I did it